trigger/virtcontainers/vc.go

```diff
@@ -9,13 +9,15 @@ package main
 import (
 	"flag"
 	"fmt"
-	"github.com/fsnotify/fsnotify"
-	"github.com/kata-containers/ksm-throttler/pkg/client"
-	"github.com/sirupsen/logrus"
 	"os"
 	"path"
 	"path/filepath"
 	"sync"
+	"time"
```

Comment:
I thought our static linters would have picked this up - is this a sign maybe that they are not running on this repo or that we need to add a 'go fmt' check or similar?

Reply:
Actually no, the central static checks are being run for this repo. That import was added because of the new reference to `time.RFC3339Nano` on this PR.

Reply:
No no, not the new addition, the moving of the URL prefixed items down to the bottom of the list... I though `go fmt` picked that up - maybe I'm wrong though.

Reply:
It's odd but `gofmt` doesn't seem to re-order this import list until the `time` gets added. Maybe there is a a heuristic to only re-order when the list gets to a certain size?